appveyor.yml

```diff
@@ -1,33 +1,45 @@
-build: false
+# https://ci.appveyor.com/project/goanpeca/pyswmm
+
+matrix:
+  fast_finish: true
+
+branches:
+  only:
+    - master 
 
 environment:
+  global:
+    # Used by qthelpers to close widgets after a defined time
+    TEST_CI: True
+    # Environment variables used by astropy helpers
+    PYTHON: "C:\\conda"
+    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci-helpers\\appveyor\\windows_sdk.cmd"
+    PYTHON_ARCH: "64" # needs to be set for CMD_IN_ENV to succeed. If a mix
```

Comment:
its irrelevant for this PR, when the tests PR is enabled we can check that.

If you are right, not being able to run on 64 is a huge bummer :-\ if that is the case. Very few users use 32 bit python.